test/specs/pushAndPull.e2e.ts

```diff
@@ -94,70 +97,67 @@ describe('Push and Pull', async () => {
       10
     );
 
-    expect(outputPanelText).toContain(
+    await expect(outputPanelText).toContain(
       `Local Add  ExampleApexClass1  ApexClass  ${path.join('force-app', 'main', 'default', 'classes', 'ExampleApexClass1.cls')}`
     );
-    expect(outputPanelText).toContain(
+    await expect(outputPanelText).toContain(
       `Local Add  ExampleApexClass1  ApexClass  ${path.join('force-app', 'main', 'default', 'classes', 'ExampleApexClass1.cls-meta.xml')}`
     );
   });
 
   step('Push the Apex class', async () => {
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org', 5);
+    const workbench = await utilities.getWorkbench();
+    await utilities.executeQuickPick('SFDX: Push Source to Default Org', Duration.seconds(5));
 
     // At this point there should be no conflicts since this is a new class.
     await verifyPushSuccess(workbench);
 
-      // Check the output.
+    // Check the output.
     await verifyPushAndPullOutputText(workbench, 'Push', 'to', 'Created');
   });
 
   step('Push again (with no changes)', async () => {
     // Clear the Output view first.
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 2);
+    const workbench = await utilities.getWorkbench();
+    await utilities.clearOutputView(Duration.seconds(2));
 
     // Now push
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org', 5);
-
-    await verifyPushSuccess(workbench);
+    await utilities.executeQuickPick('SFDX: Push Source to Default Org', Duration.seconds(5));
 
     // Check the output.
-    await verifyPushAndPullOutputText(workbench, 'Push', 'to');
+    await verifyPushSuccess(workbench);
   });
 
   step('Modify the file and push the changes', async () => {
-    const workbench = await (await browser.getWorkbench()).wait();
+    const workbench = await utilities.getWorkbench();
 
     // Clear the Output view first.
-    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 2);
+    await utilities.clearOutputView(Duration.seconds(2));
 
     // Modify the file by adding a comment.
     const textEditor = await utilities.getTextEditor(workbench, 'ExampleApexClass1.cls');
     await textEditor.setTextAtLine(3, '        // sample comment');
 
     // Push the file.
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org', 5);
+    await utilities.executeQuickPick('SFDX: Push Source to Default Org', Duration.seconds(5));
 
     await verifyPushSuccess(workbench);
 
-    await verifyPushAndPullOutputText(workbench, 'Push', 'to');
```

Comment:
Same here - why is the validation removed?

Reply:
Is this actually meant to be removed?  I see that this one is the only `verifyPushAndPullOutputText()` that is missing now.